<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionUpgrade"/>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="actionWeb"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUpgrade">
   <property name="text">
    <string>更新说明(&amp;U)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于软件(&amp;A)</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助(&amp;H)</string>
   </property>
  </action>
  <action name="actionWeb">
   <property name="text">
    <string>官网(&amp;W)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
